### PR TITLE
changed the 'm' emoji

### PR DESCRIPTION
**Pull request by empetrone**
Replaced the m emoji with a new more appropriate one
- Ⓜ instead of 〽

### CODE REVIEW

**Comment by gsora** [comment]
Hm, interesting change.

@Ludo237 @smlb01 what do you think!?

**Comment by ludo237** [comment]
It's a difficult choice, but I would go further and ask you why was 6 afraid of 7?

Six hasn't been the same since he left Vietnam. He can seldom close his eyes without opening them again at fear of Charlies lurking in the jungle trees. Not that you could ever see the bastards, mind you. They were swift, and they knew their way around the jungle like nothing else. He remembers the looks on the boys' faces as he walked into that village and... oh, Jesus. The memories seldom left him, either. Sometimes he'd reminisce - even hear - Tex's southern drawl. He remembers the smell of Brooklyn's cigarettes like nothing else. He always kept a pack of Lucky's with him. The boys are gone, now. He knows that; it's just that he forgets, sometimes. And, every now and then, the way that seven looks at him with avid concern in his eyes... it makes him think. Sets him on edge. Makes him feel like he's back there... in the jungle.

So for me the new M should not be afraid of the new M

**Comment by smlb** [comment]
Personally I think it's a #NOTABUG

**Comment by empetrone** [comment]
ni🅱️🅱️as

File: nibber/unleash.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ var Emojis = map[string]string{
 	"ngul":    "\xF0\x9F\x86\x96 \xF0\x9F\x86\x92",
 	"sos":     "\xF0\x9F\x86\x98",
 	"ok":      "\xF0\x9F\x86\x97",
-	"m":	   "\xE3\x80\xBD",
```
Comment (ludo237) [inline nibber/unleash.go]: Sorry to be bothering you but could you do without the profanities? This is a family-friendly pull request. Every Monday evening me, my grandmother and my little brother browse this repository and whatnot. It's very awkward to encounter these bad words while sitting with my family.

Comment (gsora) [reply nibber/unleash.go]: What is this profanity you're referring to?

**Pull request merged**